ms18-applicatie/Models/team-a/CalendarSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ms18_applicatie.Models.team_a
+{
+    public class CalendarSettings
+    {
+        public string ApiKey { get; set; } = string.Empty;
```

Comment:
nu voeg je een empty string als default waarde toe. misschien is een = null! beter zodat je garandeert dat het niet null is ipv een default waarde als het null is.

Reply:
je leest het in uit de secets door null! veranders je niet een uitkomst als de waarde wel of niet in de secets staat. ja kan het op 100 manieren doen. maar door null! veranderd de uitkomst niet. dit is denk puur een coding style.